src/style/themes/base/base-theme.config.js

```diff
@@ -130,6 +130,7 @@ export default (palette) => {
       tertiary: palette.slateTint(10),
       header: palette.slateTint(20),
       hover: palette.slateTint(90),
+      flatTableHover: palette.slateTint(95),
```

Comment:
shouldn't we have a separate object for the flatTable colours? Please keep in mind that we'll have multiple themes for the flat table.

Reply:
Yeah I was debating whether to do that, I wanted to avoid potentially creating conflicts for @dawidzarzycki's work on `Sort`, I'll just create a separate object for it and try to avoid creating too many conflicts for him 🤞